### PR TITLE
improving the pretrained accuracy with confusion matrix and roc curve

### PR DESCRIPTION
**Pull request by noamrud**

### CODE REVIEW

File: tools/test_pretrained_model_accuracy.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+from tools.pretrained_model_accuracy_utils import build_probs_dataframe, compute_localization_probs, evaluate_classification_accuracy, multiclass_roc
+
+import torch
+import pandas as pd
+import numpy as np
+
+def calculate_accuracy(outputs_folder_path:str, config_path_data:str, anot_file_path:str)->None:
+    config_data:DatasetConfig = load_config_file(config_path_data, 'data')
+    config_data.OUTPUTS_FOLDER = outputs_folder_path
+    config_data.MARKERS_TO_EXCLUDE = []
```
[the remark this answers is not in the document]
Comment (noamrud) [reply tools/test_pretrained_model_accuracy.py:23]: @Sagykri this behaviur appear in many of our runnables. maybe we need to think about a different way to solve this? (generate_umaps_and_plot.py, generate_embeddings.py, plot_distances.py and more...)

File: tools/test_pretrained_model_accuracy.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    probs = torch.nn.functional.softmax(outputs, dim=1).numpy()
+
+    probs_df = build_probs_dataframe(probs, labels, unique_labels, anot_file_path)
+    anot = pd.read_csv(anot_file_path)
+    sum_probs = compute_localization_probs(probs_df, anot)
+    
+    class_names = list(np.unique(anot.localization)) + ['Other']
+
+    save_path = os.path.join(outputs_folder_path,'figures', config_data.EXPERIMENT_TYPE, 'accuracy')
```
[the remark this answers is not in the document]
Comment (noamrud) [reply tools/test_pretrained_model_accuracy.py:40]: I'm using save_plot (from src.figures.plotting_utils) to plot, and this function also creating the full folder hierarchy if needed. Do you still think we need to ask of makedirs in the main script? @Sagykri
